.travis.yml

```diff
@@ -9,3 +9,18 @@ matrix:
           - docker build -t bubblesub .
           script:
           - docker run -ti bubblesub
+        - os: windows
+          services: docker
+          before_deploy:
+          - travis_wait 50 docker build -t bubblesub -f windows-build/Dockerfile .
```

Comment:
What does `travis_wait 50` mean?

Reply:
The answer to your question is [here](https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received)

I chose 50 minutes because the building process takes about 40 minutes, so I've decided to give more time to the process to avoid sudden breaks

Reply:
But it seems to be required only for commands that produce no output for 10+ minutes, whereas this command produces lots of output without any gaps?

Reply:
Unfortunately, installing `vctools` through `Chocolatey`, it requires more than 10 minutes without any output. That's why that command has been added